services/src/main/scala/cromwell/pipeline/service/ProjectFileService.scala

```diff
@@ -1,9 +1,11 @@
 package cromwell.pipeline.service
 
+import java.nio.file.Path
```

Comment:
Were there any changes to the formatter policies? My fmt attempts to revert this change

Reply:
I don't think we have such settings and don't know why Intellij decided to change it :) Rolled back to how it was before.